sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2560,6 +2560,33 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       }
     }
   }
+
+  test("SPARK-32818: make metastore convert session level configurable") {
```

Comment:
Does this test fail w/o this patch? Looks it can pass even in master.

Reply:
> Does this test fail w/o this patch? Looks it can pass even in master.

I miss the point that SQLConf.get is related to current thread's SparkSession.